azure-webapp-maven-plugin/README.md

```diff
@@ -275,6 +275,30 @@ Property | Description
 `warFile` | Specify the war file location, optional if the war file location is: `${project.build.directory}/${project.build.finalName}.war`
 `path` | Specify context path, optional if you want to deploy to ROOT.
 
+### Deployment Slot
+In the `<deploymentSlotSetting>` element of your `pom.xml` file, you can specify which deployment slot to deploy your Web App.
+It requires you to configure an existing web app. And the deployment slot configured will be created first if it does not exist.
+
+You can specify the following properties within the `<deploymentSlotSetting>` element:
+
+Property | Required | Description
+---|---|---
+`<slotName>` | true | Specifies the slot name. Valid slot name must match this kind of regex expression: [a-zA-Z0-9-]{1, 60}. 
+`<configurationSource>` | false | The configuration source used to create a deployment slot.
+
+> Note`<slotName>` is pure slot name, not the name with webapp name as prefix in the portal.
+
+If you specify an existing deployment slot to do the deploy, `<configurationSource>` will be ignored even it's configured.
+
+Or, if you want to do the deployment slot create and deploy at the same time, you could specify configuration source as you like.
+
+`<configurationSource>` could be set to **New** to create a brand new slot without any configuration,
+or set to **Parent** to create a slot and copy the configuration from parent webapp.
+Besides, you could set it another existing deployment slot name to copy configuration from during creation.
```

Comment:
Suggest changing to -> "Besides, you could set it to the name of another existing deployment slot. It will copy the configuration of that slot during creation."

Reply:
Taken